cylc/uiserver/schema.py

```diff
@@ -291,8 +291,6 @@ async def get_elements(root, info, **kwargs):
 
     _, field_ids = process_resolver_info(root, info, kwargs)
 
-    if hasattr(kwargs, 'id'):
-        kwargs['ids'] = [kwargs.get('id')]
```

Comment:
This logic copied from cylc-flow is flawed (removing in https://github.com/cylc/cylc-flow/pull/5886)